### PR TITLE
Weather App

### PR DESCRIPTION
**Pull request by dasileker**
# weather
 This project comes after the introduction of the ES6 class keyword, ES6 modules, and webpack. The main goal of this project is for the student to show the understanding of the benefits of modularized code as well as the ability to set up a javascript application made of different modules using webpack.
## Built With

 - HTML
 - CSS
 - Javascript 
 - promise function
 - API
 - Webpack
 - Used the DOM Manipulation.
 - Images
 - Icons
 - Linters
 
## Page live Demo Link

 [Live Demo Link!](https://raw.githack.com/dasileker/weather/feature/dist/index.html)

## Feedback

 - Give A  thumb-up 👍 if you like it.

### CODE REVIEW

**Review by LamiaSristy — CHANGES_REQUESTED** [review]
# Status: Change Required. ♻️

Hello @dasileker. Congratulation on your good performance in this project. There are plenty of nice features you have added here, which is making the app user-friendly. 

## The highlights: 
* The changing images with weather descriptions are very impressive.
* The Date, Months, and Year showing is good.
* I like the gif file for the background.

 ## What to do next:  
To achieve mastery you must implement some changes to the project. I have attached some feedback about the project below, which will be required.
##
**[To Improve]**
- [x] The project explicitly uses ES6 syntax, arrow function.[ScreenShot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/es6_requirement.png) from [pathwright](https://microverse.pathwright.com/library/javascript-module-v10-copy/166786/path/step/97635172/)
- [x] It has a toggle control to determine if the result will be shown in Celsius or Fahrenheit. [Odin](https://www.theodinproject.com/courses/javascript/lessons/weather-app).
toggle

- [x] The project should has error handling for `promises` or `async/await` [Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/error_handling_requirement.png) from [pathwright](https://microverse.pathwright.com/library/javascript-module-v10-copy/166786/path/step/97635172/).
- [x] In your `README` add instructions for running your project locally, `npm install` && `npm run dev`.
##
Please make the changes and ask for a re-review.

Feel free to reach me on slack @lamiasristy 🙋

Keep up the spirit always. Happy Coding 💻

**Comment by dasileker** [comment]
done.

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import showWeatherReport from './weather';
+
+const weatherApi = {
+  key: '2ac2a201f9a298fd6ce2b569bf302448',
+};
+
+const message = document.querySelector('#error');
+message.innerHTML = '';
+
+async function getReport(city) {
```
Comment (Forison) [inline src/index.js]: `const getReport= async(city) =>{`

**Review by Forison — CHANGES_REQUESTED** [review]
# Status:: Changes required❌

# 👋  Amine 

#### Good job on addressing the issue with the use of ES6 syntax, and using error handling block to handle your API requests, however here are a few recommendation we failed to address😊.

* [x] One method was define not defined using ES6 syntax.

* [x] You did not add explicitly how to start the application in the readme as requested earlier. Note not everyone that may visit your repo may be savvy as to what to do exactly, let simplify for non-technical people as stated earlier. use the command. ie
- npm i 
- npm run dev 
or perhaps tell the to 
- `cd appName, 
- `open index.html`
> Use whichever way you want but be explicit in your command and steps.

* [x] The toggle button should change the current temperature value from degree Celsius to Fahrenheit vice versa. Current the button toggles but the temperature value does not change

#### That is it, kindly make the changes and submit a re-review😎

#### Happy coding👨‍💻

Addo Forison

**Comment by dasileker** [comment]
the toggle btn is working in localhost server but nit in the live version? im trying to solve the issue

**Comment by dasileker** [comment]
All the requirement now is set like TSE's requested thank you all for the reviews.

**Review by addod19 — APPROVED** [review]
## Status: Approved :trophy: :balloon: 

Hi @dasileker, congratulations on your hard work on this project :muscle: All our functions use ES6 arrow syntax :+1: The toggle functionality now works as expected :smile: The README file has instructions about how to clone and run the app locally :slightly_smiling_face: 
I hereby approved the project :fireworks: 

[Optional]
- Let us kindly check for typo in our README file, for example; "you" instead of "yoy"


Kindly go ahead and merge into development :+1: Do not forget to submit the project completion form
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Pull request merged**